### ISSUE TITLE
OS X Lion (10.7.5): Crashes

### ISSUE THREAD

**Issue by Letterus** [pbek/QOwnNotes#264]
There seems to be a problem of Qt 5 in combination with OS X Lion 10.7.5. During startup QOwnNotes crashes. The crashlog is attached. An older version - I cannot recover which - worked on 10.7.5.
[QOwnNotes-crash.txt](https://github.com/pbek/QOwnNotes/files/385995/QOwnNotes-crash.txt)

Edit: It is version 16.07.12 that crashes.

**Comment by pbek** [comment]
Do you vaguely remember when you last did an update of QOwnNotes? Maybe you can recover that version from <https://github.com/pbek/QOwnNotes/releases>.

And can you open `~/Library/Preferences/com.pbe.QOwnNotes.plist` and try to set **networking/ignoreSSLErrors** to `true`, maybe you system has problems with some SSL certs...

**Comment by pbek** [comment]
We will try to ignore some more SSL errors in v16.07.13, let's see if that helps on your system.

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if the new features work for you?

**Comment by Letterus** [comment]
Thanks for investigating and updating! I tried to start it, but it still crashes. I tried some older releases randomly. The first one I found that worked is 16.05.11.

**Comment by pbek** [comment]
Do you still get the same stack-trace?
I've no idea what could cause the crash, the lines in the stack trace, that are belonging to QOwnNotes were added a long time before 16.05.11...

Do you care to try to build QOwnNotes yourself on that machine?
http://www.qownnotes.org/installation#building

**Comment by Letterus** [comment]
Hm, sound interesting. ;) But I've no time to test building in the moment. To me it seems that a newer version of Qt is linked against a OS X dylib containing a new function that does not exist on 10.7.5. So maybe building on this system might solve it, but maybe it would not even build?

**Comment by pbek** [comment]
> dylib

That could be the case, I guess you should give it a try...
Please tell me how it worked if you do.

**Comment by Letterus** [comment]
I am still experiencing this problem:

[crash.log.txt](https://github.com/pbek/QOwnNotes/files/611550/crash.log.txt)

This is because the SSL implementation of Qt 5 is not compatible with that of 10.7, as I found out.
Is there a way to turn off the call of the sendVisit() methods via settings file before I start the app? This way I would maybe be able to start and use QOwnNotes on this old machine even if SSL will not work.

**Comment by pbek** [comment]
You would need to turn off the tracking and metrics. But I have no way of testing QOwnNotes under macOS 10.7...

**Comment by pbek** [comment]
**appMetrics/disableAppHeartbeat**: `true`
**appMetrics/disableTracking**: `true`

**Comment by pbek** [comment]
Or is it the first time it started?

**Comment by Letterus** [comment]
Nope. Ok, I disabled these settings using my working version. But then it crashes when checking for updates. And it's no good solution to disable checking for updates. :)

I just tried this: I copied the Qt Framework from my working copy of the appliction to the application bundle of a current version: It starts! Don't know if all features will work, but that seems to be a workaround without the need to compile QOwnNotes myself.
